apple/swift-docc-render
# Use the single-letter icon for namespace symbols in navigator

Bug/issue #, if applicable: 120780967

## Summary

Updates configuration so that pages for C++ "namespace" symbols are rendered with a single-letter "N" icon in the navigator sidebar instead of a generic "article" icon.

**Example**

<img width="385" alt="Screenshot 2024-02-15 at 12 46 37 PM" src="https://github.com/apple/swift-docc-render/assets/212918/f867319f-2f1e-41a1-834f-f55ee1d2b16f">

## Dependencies

apple/swift-docc#817

## Testing

Steps:
1. Checkout this branch of swift-docc-render and build it with `npm run build`
2. Checkout my branch of swift-docc from apple/swift-docc#817 and build it with `swift build`
3. Preview the example C++ doc catalog with `DOCC_HTML_DIR=/path/to/swift-docc-render/dist swift run docc preview Tests/SwiftDocCTests/Test\ Bundles/CxxSymbols.docc`
4. Open http://localhost:8080/documentation/cxxsymbols and verify that the `Foo` namespace in the navigator is using the "N" icon

## Checklist

Make sure you check off the following items. If they cannot be completed, provide a reason.

- [X] Added tests (unit test for icon mappings are dynamically generated)

\cc @daniel-grumberg
